recipe/meta.yaml

```diff
@@ -6,7 +6,7 @@ package:
 
 build:
   number: {{ number }}
-  skip: true  # [osx or cuda_compiler_version == "None"]
```

Comment:
Just to add this is fine since [we always include `None` (even on unsupported platforms)]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/65c8727a6f32f7e713009efcc7e6ae79a49001ba/recipe/conda_build_config.yaml#L41-L42 ).